Reorder output tables

Willenst opened the pull request:
Hi, I think it's a good idea to reorder the `reason` and `desired_val` tables, because `desired_val` is mostly unpredictable and there can be really long option names, for exaple cmdline parameter `lockdown=confidentiality` or `efi=disable_early_pci_dma` would tear up the output table like this:

![image](https://github.com/user-attachments/assets/9b89a120-6924-4156-b494-7f4089fec348)

After reordering the tables it would look a bit better:

![image](https://github.com/user-attachments/assets/def87443-4b79-4854-af21-c859a8fccdb1)

 By the way, having the desired value close to the check result seems a bit more convenient :)
 
 I hope you will find this addition as a great one, waiting for your review!

a13xp0p0v commented:
Hello @Willenst,

Nice, I like this idea.

Could you please rebase this branch on the fresh `master` and also update the README?

Thank you!

a13xp0p0v commented:
I would also ask you to make the `option_name` column smaller by 2 symbols.

That will make the table look much better in the README.

Willenst commented:
Hello, I reduced the `option_name` by 2 characters:
https://github.com/a13xp0p0v/kernel-hardening-checker/pull/183/commits/6608130b270f70304afd8e0d4bbd042c8d8e11ef

And also updated the README:
https://github.com/a13xp0p0v/kernel-hardening-checker/pull/183/commits/a0a9e578ed4b7a88ee0884428574c13a2788ba8b

Rebased on a fresh master and haven't squashed commits yet to simplify the review

a13xp0p0v commented:
Applied, thanks @Willenst!
